R/dvhydrograph-render.R

```diff
@@ -293,6 +295,9 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", maxIvLa
     groundWaterLevels = list(
       points = append(list(x=x, y=y, legend.name="Measured Water Level (GWSI)"), styles$gw_level_points)
     ),
+    waterQualityData = list(
```

Comment:
Will the NULL case break this or was there already a null check somewhere not part of the PR?